torch/csrc/jit/script/compiler.cpp

```diff
@@ -312,9 +307,18 @@ std::shared_ptr<SugaredValue> emitBuiltinCall(
     if(first->type()->kind() != TupleType::Kind) {
       throw ErrorReport(loc) << "expected a tuple";
     }
-    if(inputs.size() + attributes.size() > 2) {
-      throw ErrorReport(loc) << "expected at most 2 inputs";
+
+    if(attributes.size() == 1) {
+      if(inputs.size() > 1) {
+        throw ErrorReport(loc) << "expected 1 input";
+      }
+    } else {
+      JIT_ASSERT(attributes.size() == 0);
```

Comment:
This should be an `ErrorReport` and not an assert

Reply:
findTensorOp already verified that there are no additional attributes, so that error will never happen. I added a note.